waiter/integration/waiter/basic_test.clj

```diff
@@ -468,43 +463,38 @@
       (is (str/includes? set-cookie "HttpOnly=true"))
       (is (= (System/getProperty "user.name") (str body))))))
 
-; Marked explicit due to:
-;   FAIL in (test-killed-instances)
-;   Not intersection between used and killed instances!
-;   expected: (not-empty (set/intersection all-instance-ids killed-instance-ids))
-;     actual: (not (not-empty #{}))
-(deftest ^:parallel ^:integration-slow ^:explicit test-killed-instances
+(deftest ^:parallel ^:integration-slow ^:dev test-killed-instances
```

Comment:
Why `^dev`?

Reply:
Removed the `:dev`, I was using the tag to run only these tests on my parent branch that fixes all the explicit tests.